### PR TITLE
Add _quote and _unquote to http.cookies stub

### LINKED ISSUE #4895: 'http.cookies' has no attribute '_quote'
```python
from http.cookies import _quote, _unquote
```

mypy complains that `'http.cookies' has no attribute ...` for both _quote, _unquote
pyright also do not recognize it.

### PR DESCRIPTION
**Pull request by UncleGoogle**
Resolves #4895

### CODE REVIEW

**Review by srittau — APPROVED** [review]
Thanks.

File: stdlib/3/http/cookies.pyi [hunk 1]
```diff
@@ -4,6 +4,9 @@ from typing import Any, Dict, Generic, List, Mapping, Optional, TypeVar, Union
 _DataType = Union[str, Mapping[str, Union[str, Morsel[Any]]]]
 _T = TypeVar("_T")
 
+def _quote(str: str) -> str: ...
+def _unquote(str: str) -> str: ...
```
Comment (Akuli) [inline stdlib/3/http/cookies.pyi:8]:
```suggestion
@overload
def _quote(str: None) -> None: ...
@overload
def _quote(str: str) -> str: ...
@overload
def _unquote(str: None) -> None: ...
@overload
def _unquote(str: str) -> str: ...
```

**Review by Akuli — CHANGES_REQUESTED** [review]
If the `str` variable is None, then these functions return it unchanged, but otherwise these never return None. Maybe we could do something like this to fix that?

File: stdlib/3/http/cookies.pyi (hunk 1 above)
Comment (Akuli) [reply stdlib/3/http/cookies.pyi:8]: I guess I might as well make a new PR to fix this.

**Comment by UncleGoogle** [comment]
Thanks, I didn't noticed.

**Pull request merged**